src/sbl_filing_api/entities/models/dto.py

```diff
@@ -7,10 +7,10 @@
 
 class UserActionDTO(BaseModel):
     id: int | None = None
-    user_id: str
-    user_name: str
-    user_email: str
-    timestamp: datetime
+    user_id: str | None = Field(None, max_length=36)
+    user_name: str | None = Field(None, max_length=255)
+    user_email: str | None = Field(None, max_length=255)
```

Comment:
these shouldn't be optional, you can use `...` in place of `None` to indicate required fields.